vue/components/ui/molecules/filter/filter.vue

```diff
@@ -109,6 +110,9 @@ export const Filter = {
         useQuery: {
             type: Boolean,
             default: false
+        },
+        lineupNrColumns: {
+            type: Number

```

Comment:
Either `required: true` or `default: something`. In this case let's go with `default: null`, delegating to the lineup whatever the default should be.

Reply:
Same as said before

> Setting it default: null will make the prop in lineup be null. That's why I didn't added a default to the "lineupNrColumns" prop, as I wanted the default value of "nrColumns" in "lineup" component be controlled by itself.